Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -339,19 +306,6 @@ private class NpmPathProvider : INpmPathProvider {
                 }
             }
 
-            string message;
-            if (e.WithErrors) {
-                message = SR.GetString(
-                    e.Cancelled ? SR.NpmCancelledWithErrors : SR.NpmCompletedWithErrors,
-                    e.CommandText
-                );
-            } else if (e.Cancelled) {
-                message = SR.GetString(SR.NpmCancelled, e.CommandText);
-            } else {
-                message = SR.GetString(SR.NpmSuccessfullyCompleted, e.CommandText);
-            }
-
-            ForceUpdateStatusBarWithNpmActivitySafe(message);

```

Comment:
Ok, sounds good. I misunderstood the fix the bug was asking for.

Will send out an updated pull request shortly.